docs/en/05_Contributing/16_Maintainer_Guidelines.md

```diff
@@ -0,0 +1,129 @@
+ * More straightforward changes (e.g. documentation, styling) or areas which require quite specialised expertise (e.g. React) that's less available through most Core Committers can be approved or merged by team members who aren't Core Committers
+ * Self-merges should be avoided, but are preferable to having work go stale or forcing other team members to waste time by context switching into a complex review (e.g. because the original reviewer went on leave). Any self-merge should be accompanied by a comment why this couldn't be handled in another way, and a (preferably written) approval from another team member.
+
+This role may be granted by any Core Committer,
+who should give other Core Committers an opportunity to weigh in on the decision.
+
+
+### Triage
+
+Triage of issues and pull request is an important activity: Reviewing issues, adding labels,
+closing stale issues, etc. This does not require write access to the repository as a "Contributing Committer".
+This is a great way for active community members to help out, and start a path towards becoming a Core Committer.
+
+Triage roles may be granted by any Core Committer,
+who should give other Core Committers an opportunity to weigh in on the decision.
+
+
+## The Guidelines
+
+With great power (write access) comes great responsibility.
+
+First and foremost rule of a maintainer is to collaborate with other maintainers. Follow the house rules and remember that others also care about the modules they maintain.
+
+Remember to follow the [House rules of the Core Committer team](#house-rules-of-the-core-committer-team) and the supported module [guidelines](https://www.silverstripe.org/software/addons/supported-modules-definition/).
+
+[warning]
+Some of the Supported Modules have their own house rules (usually explained in the repository README or CONTRIBUTING files). Please remember to read those carefully.
+[/warning]
+
+
+### How to triage
+
+Triaging GitHub issues and pull requests imply responding to issue/pr creators with following closing or labelling. Labelling allows the other maintainers to pick up proper issues/prs accordingly to their dedication or available resources.
+
+It is also the main way to escalate critical and high impact issues, give feedback to the community, or peer review a PR and let other maintainers know it is “good to be merged on green CI tests”.
+
+
+How to do it
+  - Regularly check the repository for new issues
+  - Ideally subscribe to the repository via [GitHub Watch](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/viewing-your-subscriptions#configuring-your-watch-settings-for-an-individual-repository) functionality
+  - Read the docs about [how we use labels](./code/#labels)
+  - If unsure about anything, it’s usually good to ask other maintainers for their opinions (on Slack or via mentioning them directly on GitHub)
+
```

Comment:
Is it worth adding reference to the 'Untriaged' tab at https://silverstripe-github-issues.now.sh/?# ?

Reply:
Done above